units/src/fee.rs

```diff
@@ -348,7 +345,7 @@ mod tests {
     #[test]
     fn fee_wu() {
         let fee_overflow = FeeRate::from_sat_per_kwu(10).to_fee(Weight::MAX);
-        assert!(fee_overflow.is_none());
+        assert!(fee_overflow.is_error());
```

Comment:
Perhaps we should test that this is mul error.

Reply:
That's a good idea and I thought of that too but at the time couldn't find the api thingy I needed.  Looks like `pub fn operation(self) -> MathOp { self.0 }` should do it on second look.

Reply:
see https://github.com/rust-bitcoin/rust-bitcoin/pull/4473